src/Monitoring/Monitoring.ArcadeServices/dashboard/arcade-services/quota.dashboard.json

```diff
@@ -1096,6 +1189,26 @@
         "query": "cores,standardDv3Family,StandardDiskCount,PremiumDiskCount,StandardSnapshotCount",
         "skipUrlSync": false,
         "type": "custom"
+      },
+      {
+        "current": {
+          "selected": false,
+          "text": "\"cab65fc3-d077-467d-931f-3932eabf36d3\",\"68672ab8-de0c-40f1-8d1b-ffb20bd62c0f\"",
```

Comment:
How is the difference between the Grafana staging and production environments handled? Do the other, non-Helix subscriptions have a staging environment? For the Helix subscriptions, specifying them both here breaks the division between staging and production that is maintained everywhere else (we'll see both instances' data in both instances' reporting). 

I see that there is data available for Arcade and XAML subscriptions in App Insights. Why are those not here?

I don't love hard-coding this data. I'd really prefer a way to get these values from something closer to the source of truth. Is there no place it can be queried? Even getting it from App Insights might work.

Reply:
The other non-Helix subscriptions doesn't have a staging environment because when they are on staging they lives on helix subscription.

When you said that there is data available for Arcade and Xaml are we talking about staging? could you please share that with me, so I can see what I am missing? 

I agree with you with no hard-coding this but now I think that there are two things here, first is that on those alerts (I think) the purpose is monitor only helix subscriptions (or is what has been monitoring since it was created) so I tried to exclude the non-helix and I created that variable (with only helix subscriptions) but you are right with your previous comment, the alert isn't going to work with the variable so I need to have a different approach.

Reply:
> When you said that there is data available for Arcade and Xaml are we talking about staging? could you please share that with me, so I can see what I am missing?

Sorry, I mean I just see it in prod. Staging has only the helix staging subscription, as you said.

Reply:
@garath I addressed the feedback of our offline conversation, could you please take a look?